back-end/slothstagram/urls.py

```diff
@@ -25,6 +10,6 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('', include('photo_app.urls')),
+    path('photo_app/', include('photo_app.urls')),
```

Comment:
Do we want to change the app URL to be in `photo_app/` instead of `/`? This seems like an unrelated change